### ISSUE TITLE
Content base rtsp rtsp 

### ISSUE THREAD

**Issue by yurgh** [AlexxIT/go2rtc#208]
Two of my twelve cameras don't work in beta frigate beta5. I get an I/O timeout in the log. Both cameras return an extra rtsp:// in Content-Base: and I thought that might be the reason. Can go2rtc sanitize the uri returned by the device?
```
Content-Base: rtsp://rtsp://turret2-cam.lan:554/stream1/
```
Trace:
```
2023-01-25 20:08:40.037310301  20:08:40.037 TRC [rtsp] client request:
2023-01-25 20:08:40.037312943  DESCRIBE rtsp://turret2-cam.lan:554/stream1 RTSP/1.0
2023-01-25 20:08:40.037315521  Require: www.onvif.org/ver20/backchannel
2023-01-25 20:08:40.037318047  CSeq: 2
2023-01-25 20:08:40.037320720  Authorization: Basic ZnJpZ2F0ZToxVmVyeVNlY3JldFB3
2023-01-25 20:08:40.037323155  Accept: application/sdp
2023-01-25 20:08:40.037325531  
2023-01-25 20:08:40.037327709  
2023-01-25 20:08:40.047809245  20:08:40.047 TRC [rtsp] client response:
2023-01-25 20:08:40.047813635  RTSP/1.0 200 OK
2023-01-25 20:08:40.047818079  Content-Length: 857
2023-01-25 20:08:40.047822906  Content-Type: application/sdp
2023-01-25 20:08:40.047828370  Content-Base: rtsp://rtsp://turret2-cam.lan:554/stream1/
2023-01-25 20:08:40.047833163  Cseq: 2
2023-01-25 20:08:40.047838032  
2023-01-25 20:08:40.047842739  v=0
2023-01-25 20:08:40.047846991  o=- 1109162014219182 1109162014219192 IN IP4 x.y.z.w
2023-01-25 20:08:40.047850351  s=RTSP/RTP stream from icamra ipc
2023-01-25 20:08:40.047853497  e=NONE
2023-01-25 20:08:40.047857190  c=IN IP4 0.0.0.0
2023-01-25 20:08:40.047861133  a=tool:LIVE555 Streaming Media v2011.05.25 CHAM.LI@ANJVISION.COM
2023-01-25 20:08:40.047864565  t=0 0
2023-01-25 20:08:40.047867867  a=range:npt=0-
2023-01-25 20:08:40.047871203  a=control:*
2023-01-25 20:08:40.047874547  m=video 0 RTP/AVP 96
2023-01-25 20:08:40.047877756  a=rtpmap:96 H264/90000
2023-01-25 20:08:40.047880955  a=control:trackID=1
2023-01-25 20:08:40.047885454  a=fmtp:96 profile-level-id=4D0016;packetization-mode=0;sprop-parameter-sets=Z00AFukBYCTQgAAB9IAAE40C,aOqPIA==;config=00000001674d0016e9016024d0800001f48000138d020000000168ea8f20
2023-01-25 20:08:40.047888576  a=x-dimensions: 704, 576
2023-01-25 20:08:40.047892004  a=x-framerate: 5
2023-01-25 20:08:40.047895421  m=audio 0 RTP/AVP 0
2023-01-25 20:08:40.047898642  a=rtpmap:0 MPEG4-GENERIC/16000/2
2023-01-25 20:08:40.047901840  a=fmtp:0 config=1408
2023-01-25 20:08:40.047905353  a=control:trackID=2
2023-01-25 20:08:40.047909278  a=Media_header:MEDIAINFO=494D4B48010100000400010010710110401F000000FA000000000000000000000000000000000000;
2023-01-25 20:08:40.047912653  a=appversion:1.0
2023-01-25 20:08:40.047920205  m=audio 0 RTP/AVP 0
2023-01-25 20:08:40.047923687  a=control:rtsp://rtsp://turret2-cam.lan:554/stream1/backchannel
2023-01-25 20:08:40.047927135  a=rtpmap:0 PCMU/16000
2023-01-25 20:08:40.047930284  a=sendonly
```

**Comment by NickM-27** [comment]
You need to show your config

**Comment by yurgh** [comment]
The minimal config
```
mqtt:
  host: 10.0.0.80

go2rtc:
  log:
    level: trace
    
  streams:
    Veranda:
      - rtsp://frigate:1VerySecretPw@turret2-cam.lan:554/stream0

cameras:
  Veranda:
    ffmpeg: 
      inputs:
        - path: rtsp://127.0.0.1:8554/Veranda?video=copy
          roles:
            - record
 

record:
  enabled: true

snapshots:
  enabled: true
  timestamp: true

ffmpeg:
  hwaccel_args: -c:v h264_qsv
  input_args: preset-rtsp-generic 
  output_args:
    record: preset-record-generic-audio-copy

detectors:
  coral:
    type: edgetpu
    device: usb
```

**Comment by AlexxIT** [comment]
Fixed. You can try go2rtc addon master version before next release

**Comment by AlexxIT** [comment]
Fixed in v1.1.0
